src/utils.js

```diff
@@ -13,8 +13,13 @@ export const getComponentVariant = (theme, componentName, variant) => {
 
 export const getComponentStyle = componentName => themeGet(`components.${componentName}.style`);
 
-export const getComponentClassName = ({ className, theme: { classPrefix }, variant }, name) =>
-  `${className || ''} ${classPrefix}-${name} ${variant ? `${classPrefix}-${name}-${variant}` : ''}`.trim();
+const getComponentClassName = ({ className, passedClassName, theme: { classPrefix }, variant }, name) =>
```

Comment:
I'm not sure why, but removing `className` from here fixed the duplication issue for me locally, while retaining the `className` on the element.

Reply:
can you get the test snapshots to pass? I've noticed the `re-box` class not being applied to `Card` when removing `className` from the props here.

Reply:
I'll check.

Reply:
Also experiencing the same failure.

Reply:
Perhaps a short-term fix could be to simply dedupe the `className` string using a set or lodash `.uniq`